xbmc/video/VideoDatabase.cpp

```diff
@@ -12105,6 +12105,121 @@ void CVideoDatabase::GetDefaultVideoVersion(VideoDbContentType itemType, int dbI
   }
 }
 
+bool CVideoDatabase::FillMovieItem(std::unique_ptr<Dataset>& dataset, int movieId, CFileItem& item)
+{
```

Comment:
Could the name reflect why this logic is not included in GetDetailsForMovie and has to be separate?

Am I on the right track with something like FillMovieItemAsVersion() or FillMovieItemAsAsset()?
Or maybe remove the call to GetDetailsForMovie() to make this a file item conversion function, like MovieItemToVersion()?